Update to support PSAP notebook scale test

kpouget opened the pull request:
This PR updates various parts of `tests/Resources/Page/ODH/ODHDashboard` to better support PSAP notebook performance scale test.

Let's find out the best way to merge my modifications in the upstream repo :)

lugi0 reviewed (approved):
Waiting on Berto's approval before merging

bdattoma reviewed (dismissed):
before merging, let's test on Jenkins against v1.24

bdattoma commented:
testing Smoke suite on Jenkins against RHODS v1.25: [1652](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1652/console) _RUNNING_

kpouget commented:
@bdattoma I've rebased the code on top of `master`, can you trigger another smoke test?

bdattoma commented:
> @bdattoma I've rebased the code on top of `master`, can you trigger another smoke test?

Jenkins jobs:
- DS Projects suite against RHODS 1.25: [1668](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1668/) _RUNNING_
- Smoke suite against RHODS v1.25: coming soon...

jgarciao commented on ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource:
```diff
@@ -26,7 +26,6 @@ Open Data Science Projects Home Page
     SeleniumLibrary.Click Link      Data Science Projects
     Wait for RHODS Dashboard to Load    wait_for_cards=${FALSE}    expected_page=Data science projects
     SeleniumLibrary.Wait Until Page Contains    View your existing projects or create new projects.    timeout=30
-    Maybe Wait For Dashboard Loading Spinner Page
```
@bdattoma is this correct?

bdattoma replied:
hm I'm not 100% sure, I would like to see it in multiple runs

jgarciao reviewed (approved):
I've added a question to Berto but LGTM

kpouget commented:
I've rebased the code against `master` and fixed some last details that were crashing the scale test.

> Maybe Wait For Dashboard Loading Spinner Page

for those ... maybe I can remove the commit 4d528db which introduces them, and we merge the rest?

the reason why I need them out when I run the scale test is that have an unconditional `sleep` that made false positive where we thought RHODS notebooks launch time increased on 10s ...

bdattoma commented:
> I've rebased the code against `master` and fixed some last details that were crashing the scale test.
> 
> > Maybe Wait For Dashboard Loading Spinner Page
> 
> for those ... maybe I can remove the commit [4d528db](https://github.com/red-hat-data-services/ods-ci/commit/4d528db387477bc5ddeb403c48ad926ce23a6082) which introduces them, and we merge the rest?
> 
> the reason why I need them out when I run the scale test is that have an unconditional `sleep` that made false positive where we thought RHODS notebooks launch time increased on 10s ...

ok with removing that commit.

I will upgrade our RHODS cluster to latest RC and re-run the PR test. Thanks!

kpouget commented:
> ok with removing that commit.
I've pushed the update

I'll prepare another PR to discuss how to get this addressed in a way compatible with the scale test and your QE tests
